tests/integration/test_bayesian_optimization.py

```diff
@@ -194,6 +195,23 @@ def GPR_OPTIMIZER_PARAMS() -> Tuple[str, List[ParameterSet]]:
                 ),
                 id="TrustRegion/MinValueEntropySearch",
             ),
+            pytest.param(
+                20,
+                MultiTrustRegionBox([TREGOBox(ScaledBranin.search_space)]),
```

Comment:
It works with `batch_size > 1`, but have to of course use a vectorised acquisition function -- I have added a new test with `ParallelContinuousThompsonSampling`. My initial thoughts were what it would not be very useful, since the regions would not maintain diversity. TREGO simply uses the global dataset minimum to centre each region, so I suspected all regions would overlap.

However the mechanism to re-initialise the regions (when they are too close) works reasonably well here. The first region behaves as normal, but the other regions end up being in the opposite mode. So we have a mixture of regions in local and global modes.